pkg/northbound/gnmi/set.go

```diff
@@ -208,10 +221,41 @@ func (s *Server) buildUpdateResults(targetUpdates mapTargetUpdates,
 		}
 		networkChanges[store.ConfigName(configName)] = changeID
 	}
-
 	return updateResults, networkChanges, nil
 }
 
+func listenForDeviceResponse(changes mapNetworkChanges, target string, name store.ConfigName) (bool, error) {
+	mgr := manager.GetManager()
+	respChan, ok := mgr.Dispatcher.GetResponseListener(topocache.ID(target))
+	if !ok {
+		log.Infof("Device %s not properly registered, not waiting for southbound confirmation", target)
+		return true, nil
+	}
+	response := <-respChan
```

Comment:
I'm guessing it blicks here until something comes in on the channel? Add a comment to that effect I think

Reply:
done